### PR TITLE
HVU baselines

### PR DESCRIPTION
**Pull request by kennymckormick**
Add 2D HVU baselines for each individual tag categories.

### CODE REVIEW

File: configs/recognition/tsn/README.md [hunk 1]
```diff
@@ -123,19 +123,32 @@ In data benchmark, we compare:
...
+| [tsn_r18_1x1x8_100e_hvu_attribute_rgb](/configs/recognition/tsn/hvu/tsn_r18_1x1x8_100e_hvu_attribute_rgb.py) |  attribute   | short-side 256 |  8   | ResNet18 | ImageNet | 46.1 |   33.6    |      34.9       | [ckpt](https://download.openmmlab.com/mmaction/recognition/tsn/hvu/attribute/tsn_r18_1x1x8_100e_hvu_attribute_rgb_20201027-0b3b49d2.pth) | [log](https://download.openmmlab.com/mmaction/recognition/tsn/hvu/attribute/tsn_r18_1x1x8_100e_hvu_attribute_rgb_20201027.log) | [json](https://download.openmmlab.com/mmaction/recognition/tsn/hvu/attribute/tsn_r18_1x1x8_100e_hvu_attribute_rgb_20201027.json) |
+|                              -                               |   Overall    | short-side 256 |  -   | ResNet18 | ImageNet | 52.6 |   40.0    |      41.3       |                              -                               |                           [log]()                            |                           [json]()                           |
```
Comment (dreamerlin) [inline configs/recognition/tsn/README.md:136]: [log]() and [json]()

**Review by dreamerlin — APPROVED** [review]

**Comment by kennymckormick** [comment]
@innerlee This PR is ready now.

**Comment by innerlee** [comment]
okay

File: mmaction/core/evaluation/accuracy.py [hunk 2]
```diff
@@ -149,6 +149,7 @@ def mean_average_precision(scores, labels):
             scores[i], labels[i])
         ap = -np.sum(np.diff(recall) * np.array(precision)[:-1])
         results.append(ap)
+    results = [x for x in results if not np.isnan(x)]
```
Comment (innerlee) [inline mmaction/core/evaluation/accuracy.py:152]: curios, can results be empty?

File: configs/recognition/tsn/hvu/tsn_r18_1x1x8_100e_hvu_attribute_rgb.py [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+optimizer = dict(
+    type='SGD', lr=0.01, momentum=0.9,
+    weight_decay=0.0001)  # this lr is used for 8 gpus
+optimizer_config = dict(grad_clip=dict(max_norm=40, norm_type=2))
+# learning policy
+lr_config = dict(policy='step', step=[40, 80])
+total_epochs = 100
+checkpoint_config = dict(interval=1)
+evaluation = dict(
+    interval=2, metrics=['top_k_accuracy', 'mean_class_accuracy'], topk=(1, 5))
```
Comment (innerlee) [inline configs/recognition/tsn/hvu/tsn_r18_1x1x8_100e_hvu_attribute_rgb.py:115]: topk now should be deleted, right? @dreamerlin

**Comment by innerlee** [comment]
looks good

File: configs/recognition/tsn/hvu/tsn_r18_1x1x8_100e_hvu_attribute_rgb.py (hunk 3 above)
Comment (dreamerlin) [reply configs/recognition/tsn/hvu/tsn_r18_1x1x8_100e_hvu_attribute_rgb.py:115]: Yes, should delete ', topk=(1, 5)'

File: mmaction/core/evaluation/accuracy.py (hunk 2 above)
Comment (kennymckormick) [reply mmaction/core/evaluation/accuracy.py:152]: It happens when a class exists in the training set but doesn't exist in the testing set. (A bug in the released HVU dataset).

Comment (kennymckormick) [reply mmaction/core/evaluation/accuracy.py:152]: It happens when a class exists in the training set but doesn't exist in the testing set. [A quick fix of a bug I've written, for the HVU object class, I wrongly set the num_classes to 1679 (actually it is 1678)].  I'll fix it in the next version.

Comment (innerlee) [reply mmaction/core/evaluation/accuracy.py:152]: mean of empty is not defined. try np.mean([]) to see the warning

**Comment by innerlee** [comment]
need to check if results are empty in `mmaction/core/evaluation/accuracy.py`

**Review by innerlee — APPROVED** [review]

**Pull request merged**